lava-v2-jobs-from-api.py

```diff
@@ -96,10 +124,9 @@ def main(args):
                         exit(1)
             if build['kernel_image']:
                 # handle devices without a DTB, hacky :/
+                build['dtb_dir_data'].append('qemu')
```

Comment:
Yes the link I put in my previous comment is about the arch in the job context, I've got a fix for that.

With LAVA v1, a different device type was needed for each qemu arch.  This is not the case anymore, so I think things can be consolidated a bit better for v2.  It's been useful for me to run on qemu to test the changes I'm working on, I'll look a bit more into this and send an updated patch.